kie-api/src/main/java/org/kie/api/internal/utils/ServiceDiscoveryImpl.java

```diff
@@ -112,8 +100,10 @@ public void registerConfs( ClassLoader classLoader, URL url ) {
                     processKieService( classLoader, entry[0].trim(), entry[1].trim() );
                 }
             }
-        } catch (Exception exc) {
-            throw new RuntimeException( "Unable to build kie service url = " + url.toExternalForm(), exc );
+        } catch (RuntimeException re) {
+            throw re;
```

Comment:
What is the reason of this change? I understand that this avoid to wrap the exception but I think it makes more sense to add some contextual information like in the other catch (`"Unable to build kie service url = " + url.toExternalForm()`).
Wdyt?